website/content/en/guides/level-up/log_namespace.md

```diff
@@ -0,0 +1,188 @@
+---
+title: The Log Namespace feature
+short: Log Namespace
+description: Learn how the log namespacing works.
+author_github: https://github.com/pront
+domain: schemas
+weight: 2
+tags: ["log_namespace", "logs", "namespace", "level up", "guides", "guide"]
+---
+
+{{< requirement >}}
+Before you begin, this guide assumes the following:
+
+* You understand this feature and the global schema are mutually exclusive.
```

Comment:
Should there be more than one schema?
```suggestion
* You understand this feature, and the global schemas are mutually exclusive.
```
...or is the sentence comparing the feature and the schema?
```suggestion
* You understand that this feature and the global schema are mutually exclusive.
```